Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "grammer"
-version = "0.0.1"
+version = "0.0.2"
```

Comment:
Oh I forgot to mention, please don't change the version number, `grammer` is still unpublished.

Reply:
Done.